### PR TITLE
refactor(Core/Spell): Move some SpellImmune from script to db

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Move from core to db

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4380

## Tests Performed:
- None

## How to Test the Changes:

Not really needed as the functionality of the table was tested in https://github.com/azerothcore/azerothcore-wotlk/pull/4358

1. Cast a spell with the ignored school at the creature's face :)

## Target Branch(es):
- [x] Master

TBM after https://github.com/azerothcore/azerothcore-wotlk/pull/4358

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
@Kitzunu build failing

**Comment by Kitzunu** [comment]
image
@FrancescoBorzi

**Comment by FrancescoBorzi** [comment]
ah ok

File: data/sql/updates/pending_db_world/rev_1612059439455457600.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612059439455457600');
+
+-- SPELL_SCHOOL_FIRE
+UPDATE `creature_template` SET `spell_school_immune_mask` = 4 WHERE `entry` IN (17267,15438);
+
+-- SPELL_SCHOOL_NATURE
+UPDATE `creature_template` SET `spell_school_immune_mask` = 8 WHERE `entry` IN (30258,15352);
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1612059439455457600.sql:7]: +31463

**Comment by pklloveyou** [comment]
Mark

**Comment by pklloveyou** [comment]
GOOD

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Helias** [comment]
build fail

**Comment by Kitzunu** [comment]
Once again

**Comment by Kitzunu** [comment]
SmartSelect_20210201-105859_Chrome

**Pull request merged**